### ISSUE TITLE
Building without the Unity project path?

### ISSUE THREAD

**Issue by INeatFreak** [pCYSl5EDgo/create-unitypackage#5]
Is it possible to still build the package without the project path? I mean when the target folder and git root isn't inside of an unity project. If so, how do I make that happen. I've been trying over 2 hours to get this to work but couldn't do it.

**Comment by PhantasmicDev** [comment]
> Is it possible to still build the package without the project path? I mean when the target folder and git root isn't inside of an unity project. If so, how do I make that happen. I've been trying over 2 hours to get this to work but couldn't do it.

Did you ever get it to work? Or did you find a different solution? I'm currently trying to do the same.

**Comment by INeatFreak** [comment]
> 

Nope, I've gave up as It didn't worth my time to edit the source code to support this.

**Comment by crashkonijn** [comment]
You can achieve it like this, you should make sure you cd into the project folder as well before creating the meta files. The issue in #7 is the same.

```
      - uses: actions/checkout@v2
      - run: |
          cd Demo
          echo "Assets/CrashKonijn.meta" > metaList
          find Assets/CrashKonijn/ -name \*.meta >> metaList
          echo "Contents of metaList:"
          cat metaList

      - uses: pCYSl5EDgo/create-unitypackage@master
        with:
          package-path: 'Package/Samples~/test/demo.unitypackage'
          include-files: Demo/metaList
          project-folder: 'Demo/'
```

In this case the project folder is the `Demo` folder.

**Comment by pCYSl5EDgo** [comment]
I hope [v1.2.1](https://github.com/pCYSl5EDgo/create-unitypackage/releases/tag/v1.2.1) will help all of you.
